textBasedAdventure.js

```diff
@@ -1,5 +1,122 @@
 const readline = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+
+const play = () => {
+    let userInput 
+    while(userInput != "yes") {
+        userInput = readline.question("Would you like to play Game of Thrones, yes or no?")
+    }
+    console.log("WELCOME TO BATTLE FOR THRONES, PROTECT YOUR KINGDOM AT ALL COST !!")
+}
+
+
+const gameIntro = () => {
+    console.log("GAME INTRODUCTION")
+
+}
+
+
+const kingdomName = () => { // gets the name from the user and saves it 
+    let userInput
+    userInput = readline.question("What would you like to name your kingdom?")
+    return userInput 
+}
+
+
+
+
+console.log("option of who they would like to fight first compared to army strength")
+console.log("WHO WOULD YOU LIKE TO CONQUER FIRST!?")
+
+
+const removeConquered = (array, oppName) => {
+    let search = oppName
+    for(let i = 0; i < array.length; i++) {
+        if(array[i] === search){
+            array.splice(i, 1)
+        }
+    }
+}
+
+
+const fightOption = () => {
+    let userInput
+
+    userInput = readline.question("WHO'S KINGDOM WOULD YOU LIKE TO CONQUER!?")
+    listRivals()
+    if (userInput === "John Snow" || userInput === "john snow") {
+        console.log("YOU HAVE CHOSE TO CHALLENGE " + rivalsName(userInput + "'s KINGDOM, LET THE SHOWDOWN BEGIN !"))
+        fight1(myArmySize, jSnowArmy, userInput)
+    }else if (userInput === "K" || userInput === "k") {
+        console.log("YOU HAVE CHOSE TO CHALLENGE " + rivalsName(userInput + "'s KINGDOM, LET THE SHOWDOWN BEGIN !"))
+        fight1(myArmySize, kArmy, userInput)
+    }else if (userInput === "B" || userInput === "b") {
+        console.log("YOU HAVE CHOSE TO CHALLENGE " + rivalsName(userInput + "'s KINGDOM, LET THE SHOWDOWN BEGIN !"))
+        fight1(myArmySize, bSnowArmy, userInput)
```

Comment:
Your functions are printed instead of being invoked, `myArmySize`,  `bSnowArmy()`